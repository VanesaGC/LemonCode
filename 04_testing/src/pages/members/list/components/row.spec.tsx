import * as React from "react";
import { shallow } from "enzyme";
import { Row } from "./row";

describe("pagesmemberslistcomponents (row) tests", () => {
  it("should render as expected when passing required properties", () => {
    // Arrange
    const props = {
      member: {
        id: 1,
        name: "Mar√≠a",
        avatarUrl: "maria.png"
      }
    };
    // Act
    const component = shallow(<Row {...props} />);
    // Assert
    expect(component).toMatchSnapshot();
  });
});
