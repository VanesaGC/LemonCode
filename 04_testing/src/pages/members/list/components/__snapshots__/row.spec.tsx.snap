// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pagesmemberslistcomponents (row) tests should render as expected when is invoked 1`] = `
<tr>
  <td>
    <img
      src="maria.png"
      style={
        Object {
          "maxWidth": "10rem",
        }
      }
    />
  </td>
  <td>
    <span>
      1
    </span>
  </td>
  <td>
    <span>
      María
    </span>
  </td>
</tr>
`;

exports[`pagesmemberslistcomponents (row) tests should render as expected when passing required properties 1`] = `
<tr>
  <td>
    <img
      src="maria.png"
      style={
        Object {
          "maxWidth": "10rem",
        }
      }
    />
  </td>
  <td>
    <span>
      1
    </span>
  </td>
  <td>
    <span>
      María
    </span>
  </td>
</tr>
`;
